README.md

```diff
@@ -39,4 +39,4 @@ Drawable drawable = ImagePreview.previewDrawable(context, "Your Base64 encoded p
 
 ## License
 
-ImagePreview is available under the MIT license. See the LICENSE file for more info.
\ No newline at end of file
+ImagePreview is available under the MIT license. See the LICENSE file for more info.
```

Comment:
(Not sure what the change is here, I made this PR using github's web interface.)